icons/library-big.svg

```diff
@@ -0,0 +1,15 @@
+<svg
```

Comment:
I don't like us having too variants that have the same use case, which for those `-big` versions feels like it is usually the case.

Reply:
In general maybe, but I think this one is different enough to warrant a variant (and looks more ‘Lucide’ to me…)